src/features/game/events/plant.ts

```diff
@@ -135,6 +135,10 @@ export function plant({ state, action, createdAt = Date.now() }: Options) {
     throw new Error("Field does not exist");
   }
 
+  if (!inventory["Water Well"]) {
```

Comment:
This `plant.ts` event is used in the current game. Unfortunately this would throw an error for all current players since no one has a Water Well at this point.

Consider moving this into the `plant()` inside of `landExpansion/plant`